README.md

```diff
@@ -243,7 +243,7 @@ Alternatively, you may also directly ssh into the guest: `ssh -p 60022 -i ~/.lim
 Yes, it should work, but not regularly tested on ARM (due to lack of CI).
 
 #### "Can I run non-Ubuntu guests?"
-Alpine, Arch Linux, CentOS, Debian, Fedora, and openSUSE are also known to work.
+Alpine, Arch Linux, CentOS, Debian, Fedora, openSUSE, and Rocky are also known to work.
```

Comment:
Have you meant to remove CentOS from here as well?

Reply:
No, CentOS is still known to work.

Reply:
We deliberately removed CentOS 7 support (from lima), and there will soon be no more support for CentOS 8 (and there is no CentOS 9). So it is going to "disappear" eventually (next year), unless adding stream.yaml and explaining the difference.

Reply:
Removed "CentOS"